src/main/java/bio/terra/stairway/DbUtils.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.stairway;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+/** Database methods shared across DAOs */
+public class DbUtils {
+  static void startTransaction(Connection connection) throws SQLException {
```

Comment:
Does this class need to be public if it's functions are package private? ditto on DbRetry